src/pages/news.js

```diff
@@ -28,17 +33,26 @@ export default function NewsPage(props) {
   // console.warn('default releases:', defaultReleases)
   /* eslint-disable-next-line no-unused-vars */
   const [releases, updateReleases] = useState(oldReleases)
-  const newsItems = posts.concat(releases)
+  const [updated, setUpdated] = useState(false)
+
+  for (const release of releases) {
+    release.name = release.npm ? release.npm.name : null
+    release.type = 'release'
+    release.url = release.github.html_url
+    delete release.description
+  }
 
+  const newsItems = posts.concat(releases)
   newsItems.sort((a, b) => b.date.localeCompare(a.date))
 
-  /*
-  setTimeout(async () => {
-    const latest = await getReleases()
-    console.warn('latest releases:', latest)
-    updateReleases(latest)
-  }, 10)
-  */
+  if (!updated) {
+    setTimeout(async () => {
+      const latest = await getReleases()
+      // console.warn('latest release data:', latest)
+      updateReleases(latest)
+      setUpdated(true)
+    }, 10)
+  }
```

Comment:
Can we do this in a `useEffect` hook instead of using `setTimeout`? I think if use `useEffect` we won't need to track the `updated` state:

```js
useEffect(() => {
  getReleases().then(newReleases => setReleases(newReleases))
}, [])
```